src/models/referring/index.js

```diff
@@ -0,0 +1,25 @@
+import sql from 'sql-template-tag'
+import { app } from '../../index.js'
+import { EntryError } from '../../utils/errors.js'
+
+const insert = async (referred_profile_id, referred_by_id) => {
```

Comment:
code style use camel case  on variable and function parameter use snake case on objects